src/vulkan_graphics_1.cpp

```diff
@@ -0,0 +1,399 @@
+// Unit test to try out vulkan graphic with variations
+// Based on https://github.com/Erkaman/vulkan_minimal_compute
+
+#include "vulkan_common.h"
+#include "vulkan_graphics_common.h"
+
+// contains our compute shader, generated with:
+//   glslangValidator -V vulkan_graphics_1.vert -o vulkan_graphics_1_vert.spirv
+//   xxd -i vulkan_graphics_1_vert.spirv > vulkan_graphics_1_vert.inc
+//   glslangValidator -V vulkan_graphics_1.frag -o vulkan_graphics_1_frag.spirv
+//   xxd -i vulkan_graphics_1_frag.spirv > vulkan_graphics_1_frag.inc
+
+#include "vulkan_graphics_1_vert.inc"
+#include "vulkan_graphics_1_frag.inc"
+
+#include <glm/glm.hpp>
+#include <glm/gtc/matrix_transform.hpp>
+
+#define GLM_FORCE_RADIANS
+#define GLM_FORCE_DEPTH_ZERO_TO_ONE
+#include <chrono>
+
+#define STB_IMAGE_IMPLEMENTATION
+#include <stb_image.h>
```

Comment:
Should we add this file in external as well, like we do with stb_image_write.h now?